cv/qc-cv.obo

```diff
@@ -417,6 +417,1146 @@ is_a: QC:4000024 ! MS2 metric
+is_a: QC:4000071 ! Column type
+has_units: NCIT:C45253 ! String
+
+[Term]
+id: QC:4000076
+name: Length values
+def: "The column contains length values of sequences, i.e. the count of items within the sequences, e.g. total length of proteins." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: STATO:0000047 ! count
+
+[Term]
+id: QC:4000077
+name: Target or decoy designation
+def: "The column contains designations of the type 'target' or 'decoy'." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: NCIT:C45253 ! String
+
+[Term]
+id: QC:4000078
+name: Sample rates
+def: "The column contains sample rates as number of occurrences." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: PATO:0000161 ! rate
+
+[Term]
+id: QC:4000079
+name: Counts
+def: "The column contains counts of occurrences of a common type." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: STATO:0000047 ! count
+
+[Term]
+id: QC:4000080
+name: Peptide sequences
+def: "The column contains peptide sequences." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: NCIT:C45253 ! String
+
+[Term]
+id: QC:4000081
+name: Outliers below
+def: "The outlier values of a given type below 1.5 * IQR." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: NCIT:C79083 ! Outlier
+
+[Term]
+id: QC:4000082
+name: Outliers above
+def: "The outlier values above 1.5 * IQR." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: NCIT:C79083 ! Outlier
+
+[Term]
+id: QC:4000083
+name: Protein coverage
+def: "The coverage of protein sequences from the peptide sequences identified. The table records the coverage itself (in percent), the protein accession (SHOULD correspond to the accession used in fasta document used for Sequence DB based identification), Length of the protein(, and optionally if it is 'target' or 'decoy' entry.)" [PSI:QC]
+comment: The protein coverage can provide insight into the sensitivity of the instrument/identification/protocols used.
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column: QC:4000074 ! Sequence coverages
+has_column: QC:4000075 ! Protein accessions
+has_column: QC:4000076 ! Length values
+has_optional_column: QC:4000077 ! Target or decoy designation
+
+[Term]
+id: QC:4000084
+name: Sampling rates
+definition: "The sampling rates of identified peptides and their respective frequency (accumulated by the sampling rate, i.e. how often peptides were sampled n times and how many peptides were resampled at that rate. Optionally, which peptides were sampled at that rate, space separated)" [PSI:QC]
+comment: The sampling rate may give insights to peptide separation, dynamic exclusion settings and gradient efficiency.
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column: QC:4000078 ! Sample rates
+has_column: QC:4000079 ! Counts
+has_optional_column: QC:4000080 ! Peptide sequences
+
+[Term]
+id: QC:40000
+name: Explained precursor intensity first quarter
```

Comment:
this description is very confusing....
what exactly is computed here?
Of all MS2 scans whose precursor intensity is within the first quarter of all precursor intensities, how many were identified?

Reply:
see definition